### PR TITLE
Integrate all features to develop

### PR DESCRIPTION
**Pull request by Daramola98**
#### What does this PR do?
- Integrate all features for staging

#### Description of Task to be completed?
- Integrate all Features 

#### How should this be manually tested?
- All stories in the pivotal tracker link below should be up and running

#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/n/projects/2173231

### CODE REVIEW

File: app/controllers/users.js [hunk 1]
```diff
@@ -338,6 +339,46 @@ exports.addDonation = (req, res) => {
...
+      })
+        .exec((err, user) => {
+        // Confirm that this object hasn't already been entered
+          let duplicate = false;
+          for (let i = 0; i < user.donations.length; i++) {
+            if (user.donations[i].crowdrise_donation_id === req.body.crowdrise_donation_id) {
+              duplicate = true;
+            }
+          }
+          // if (!duplicate) {
```
Comment (fortunee) [inline app/controllers/users.js]: Remove commented out code here and every other place
@Daramola98

**Review by fortunee — COMMENTED** [review]
Hi @Daramola98 When you're done merging this to develop you raise a PR against master from develop

File: app/controllers/users.js (hunk 1 above)
Comment (Daramola98) [reply app/controllers/users.js]: Ok

Comment (Daramola98) [reply app/controllers/users.js]: Done

**Pull request merged**